invenio_rdm_records/config.py

```diff
@@ -197,6 +197,10 @@ def always_valid(identifier):
         title=_("Least recently updated"),
         fields=["updated"],
     ),
+    "mostviewed": dict(title=_("Most viewed"), fields=["-stats.all_versions.views"]),
+    "mostdownloaded": dict(
+        title=_("Most downloaded"), fields=["-stats.all_versions.downloads"]
+    ),
```

Comment:
this works if the statistics are only dumped into the record JSON before indexing (e.g. via a dumper extension), and does not require the statistics to be part of the record otherwise